### DISCUSSION TITLE
user-guide/wps-plugin

### DISCUSSION THREAD

**Discussion by giscus** [zotero-chinese/wiki#206] [文档评论区]
# user-guide/wps-plugin

Zotero 非官方中文维护小组

https://zotero-chinese.com/user-guide/wps-plugin

**Comment by HISUNDAY** [comment]
从zotero6升级到zotero7后，原本的WPS引用无法使用，使用您的方法后未WPS未出现新的扩展按钮请问如何解决

**Comment by pptt-pptt** [comment]
显示“WPS-Zotero:Network error occurred,is Zotero running?”要怎么办

**Reply by AatroxBill** [reply]
同样遇到了这个问题

**Reply by northword** [reply]
https://github.com/tankwyn/WPS-Zotero/issues/32

> 我也遇到了一样的问题，后来发现是proxy的问题。解压wps-zotero之后的那个文件夹里有个proxy.py的文件，如果是windows用户，需要手动双击打开一下那个文件，然后再在wps里使用zotero插件，可以具体看我的视频：
> https://www.youtube.com/live/L6x6d-7DUio?si=Zk2qb4Yf2wlhqgPJ&t=865

**Reply by northword** [reply]
https://github.com/tankwyn/WPS-Zotero/issues/16

> 在 Windows 上，WPS JSAPI（该插件所依赖的）非常不可靠。最新版本 WPS 的一个已知问题是wps.OAAssist.ShellExecute函数无法再启动本地程序，这会导致 [#16](https://github.com/tankwyn/WPS-Zotero/issues/16)。作为一种临时解决方案，可以通过双击手动启动 proxy.py。

**Reply by AatroxBill** [reply]
确实解决了，感谢

https://github.com/tankwyn/WPS-Zotero/issues/16#issuecomment-1875432277

**Comment by heidimyjoy** [comment]
mac系统上wps无法使用zotero插件吗？可是我office的word在mac上会闪退，已经丢了两次了！one drive也会失灵！mac系统上wps无法使用zotero插件太痛苦了！！！！

**Reply by Gmingming** [reply]
有同样的问题

**Reply by bigben446** [reply]
可以使用wangrui5015修改版本，适配最新版本的Win、linux、mac版本WPS：

https://gitee.com/wangrui5015/Zotero-WPSJS

**Comment by YJZcoder** [comment]
你好，我之前wps使用插件没有问题，但11月某次系统自动升级后，每次打开WPS中的word，均出现一个弹框，内容如下：

“您即将打开其他程序或文件，可能存在安全风险，请务必确认您完全信任程序或文件的来源，以避免可能遭受恶意软件的攻击。将打开的程序或文件：pythonw.exe。请问您是否继续？”

只有是/否的选项按钮，点击“是”才能引用文献。但每次开WPS word都会弹出这个框，很影响日常文档处理工作，请问有无方法能一次性处理掉这个警告弹框呢？

盼复，感谢！

**Reply by cassie2024gitgit** [reply]
有同样的问题，请问如何解决？每次打开都有询问，很烦啊！

**Reply by YJZcoder** [reply]
你好，我这边问题已解决，你可以试试，希望有所帮助。
1、先卸载之前的插件（双击之前文件夹内的“windows安装与卸载.bat ”文件，根据提示选择数字2卸载。
2、参考该贴上方“bigben446”的回复，使用wangrui5015修改版本https://gitee.com/wangrui5015/Zotero-WPSJS，根据说明安装。
在此，也感谢插件作者们的大力帮助

**Comment by HISUNDAY** [comment]
很抱歉现在的网络没法使用github,不知这样回复你是否能收到，我曾出现这样的问题，在一次全面的关闭电脑，关闭zotero，删除缓存后，这样的问题就消失了，你可以试一试。


| |
周严东
|
|
***@***.***
|
---- Replied Message ----
| From | ***@***.***> |
| Date | 12/16/2024 14:07 |
| To | ***@***.***> |
| Cc | ***@***.***>,
***@***.***> |
| Subject | Re: [zotero-chinese/wiki] user-guide/wps-plugin (Discussion #206) |

你好，我之前wps使用插件没有问题，但11月某次系统自动升级后，每次打开WPS中的word，均出现一个弹框，内容如下：

“您即将打开其他程序或文件，可能存在安全风险，请务必确认您完全信任程序或文件的来源，以避免可能遭受恶意软件的攻击。将打开的程序或文件：pythonw.exe。请问您是否继续？”

只有是/否的选项按钮，点击“是”才能引用文献。但每次开WPS word都会弹出这个框，很影响日常文档处理工作，请问有无方法能一次性处理掉这个警告弹框呢？

盼复，感谢！

**Reply by YJZcoder** [reply]
您好，感谢您的回复。请问如何删除缓存，电脑还是zotero的缓存呢？ 步骤是①关闭电脑，②开机、关闭zotero，③删除缓存，对吗？

**Comment by YJZcoder** [comment]
你好，感谢您的回复与支持。我在尝试安装后，出现如下错误提示，请问可以如何解决呢？ 我估计可能与我zotero安装路径（D盘）与此图中提示中默认的位置不同有关，但我不知如何修改。期待您的回复，感谢










------------------&nbsp;Original&nbsp;------------------
From:                                                                                                                        "zotero-chinese/wiki"                                                                                    ***@***.***&gt;;
Date:&nbsp;Sat, Dec 28, 2024 04:51 PM
***@***.***&gt;;
***@***.******@***.***&gt;;
Subject:&nbsp;Re: [zotero-chinese/wiki] user-guide/wps-plugin (Discussion #206)





 
可以使用wangrui5015修改版本，适配最新版本的Win、linux、mac版本WPS：
 
https://gitee.com/wangrui5015/Zotero-WPSJS

**Comment by TTTCCR** [comment]
目前MacOS最新版本的WPS是不是已经开放接口了呢？
WPS最新版本6.14.0的更新说明里面有提到“新增JSapi能力的支持，可以支持JS宏。”

**Reply by bigben446** [reply]
可以使用wangrui5015修改版本，适配最新版本的Win、linux、mac版本WPS：

https://gitee.com/wangrui5015/Zotero-WPSJS

**Comment by Mona99-star** [comment]
您好，双击windows安装与卸载.bat 安装文件后，小黑屏显示：用提供模式无法找到Python文件。但输入python --version时可以看到Python版本，请问大佬，这个该如何解决。

**Comment by Aspirefor** [comment]
这是什么问题该怎么解决啊：
Traceback (most recent call last):
  File "E:\WPS-Zotero-main\install.py", line 94, in <module>
    uninstall()
  File "E:\WPS-Zotero-main\install.py", line 84, in uninstall
    xmlStr = f.read()
             ^^^^^^^^
UnicodeDecodeError: 'gbk' codec can't decode byte 0x80 in position 161: illegal multibyte sequence

**Comment by HsiaMan** [comment]
Error occurred TypeError, please click dev tool, navigate to console and report the issue.
You will have to restart Zotero
Mac 下出现这个问题，并且点了之后卸载之后写不了。还有一个问题，就是我插入了文献之后，格式会乱

**Reply by Hr1oc** [reply]
您好，我也出现了类似的问题，有解决措施了吗

**Comment by xbbpbb** [comment]
Project-wpsif:第2行第22列:语法错误:Unexpected token=,出现这个状况是什么问题 zotero8

**Comment by christalli611** [comment]
请问显示Word could not communicate with Zotero怎么办呀，我zotero是打开的

**Comment by xbbpbb** [comment]
刚去看是正常的了，但是我有点忘记当时是怎么搞好的，重装或者重启一下zotero试试呢

________________________________
发件人: Hr1oc ***@***.***>
发送时间: 2026年4月2日 17:16
收件人: zotero-chinese/wiki ***@***.***>
抄送: xbbpbb ***@***.***>; Comment ***@***.***>
主题: Re: [zotero-chinese/wiki] user-guide/wps-plugin (Discussion #206)


您好，我也出现了类似的问题，有解决措施了吗

―
